alacritty_terminal/src/tty/windows/winpty.rs

```diff
@@ -71,9 +71,9 @@ pub fn new<C>(config: &Config<C>, size: &SizeInfo, _window_id: Option<usize>) ->
     let child_watcher = ChildExitWatcher::new(agent.raw_handle()).unwrap();
 
     Pty {
-        backend: super::PtyBackend::Winpty(agent),
-        conout: super::EventedReadablePipe::Named(conout_pipe),
-        conin: super::EventedWritablePipe::Named(conin_pipe),
+        backend: super::Backend::from(agent),
+        conout: super::ReadPipe::from(conout_pipe),
+        conin: super::WritePipe::from(conin_pipe),
```

Comment:
Is there any reason to use the renamed values here? Seems like that just increases complexity.

Reply:
I was going for consistency with `conpty.rs`, where the renamed values will map to different concrete types depending on whether the `winpty` feature is enabled.

I'll change back to using the concrete types.